clang_bind/cmake_frontend.py

```diff
@@ -0,0 +1,162 @@
+import os
+import json
+
+
+class Codemodel:
+    def __init__(self, reply_dir):
+        self.reply_dir = reply_dir
+        self.codemodel = {}
+        self.codemodel_targets = {}
+
+    def set_codemodel(self, codemodel_file=None):
```

Comment:
Thoughts on set_codemodel_from_file or read_codemodel?

Reply:
Removed the class taking into account the suggestion below.